pkg/kubelet/stats/cri_stats_provider_others.go

```diff
@@ -0,0 +1,29 @@
+// +build !windows
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package stats
+
+import (
+	statsapi "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
+)
+
+// listContainerNetworkStats returns the network stats of all the running containers.
+func (p *criStatsProvider) listContainerNetworkStats() (map[string]*statsapi.NetworkStats, error) {
+	// Always return nil for unsupported platforms.
+	return nil, nil
```

Comment:
This is going to yield a nil pointer exception on unsupported platforms on cri_stats_provider.go:166. Not sure if it matters, but maybe this could return an error like https://github.com/kubernetes/kubernetes/blob/48def74e641f8dff50fcf7c33c2a412593e2fede/pkg/util/mount/exec_mount_unsupported.go ?

Reply:
In such case, we still want to continue other stats logic in callers. So nil is returned here instead of errors.

Reply:
For Linx, there always have network stats, but for Windows, there doesn't before this PR.  After this, both Windows and Linux containers would have network stats.